src/modm/architecture/module.lb

```diff
@@ -48,13 +48,27 @@ class Assert(Module):
         module.description = FileReader("interface/assert.md")
 
     def prepare(self, module, options):
-        module.depends(":math:utils", ":architecture:accessor")
+        module.add_option(
+            BooleanOption(name="with_description",
+                          default=options[":target"].identifier.platform in ["hosted"],
+                          description=assert_include_description_descr))
+
+        module.depends(":math:utils", ":architecture:register")
```

Comment:
Was the dependency `:architecture:accessor` removed intentionally?

Reply:
Yesno. I added it back for only AVRs.